docs/quick-start/ignite-commands.md

```diff
@@ -0,0 +1,138 @@
+# Ignite Commands
+
+This document reviews the commands that the Ignite 2.0 CLI provides. This guide
+assumes that you've installed Ignite. If you haven't, then check out our Getting
+Started Guide [here]().
+
+Enter `ignite` into your command line to see the commands offered by Ignite 2.0.
+
+
+### Add
+
+```
+ignite add {ignite plugin name}
+```
+
+Adds an Ignite plugin.
+
+Ignite plugins are just node plugins. They must have an index.js file with an
+`add` function. Calling `ignite add <plugin name>` calls this function. Its job is to add the
```

Comment:
Here's the twist.

`index.js` is optional.  But if it exists, it needs to export 2 functions:  `add` and `remove`.

The reason why `index.js` is optional is because the "generator" style of plugins live in the `commands` sub-directory.  And that's all gluegun needs to piece things together.  Not sure if you have to say exactly that, but just wanted to provide more context.